### PR TITLE
Comic page

### PR DESCRIPTION
**Pull request by davidpmccormick**
## What is this PR trying to achieve?
Adding the comic article template. This is working on the assumption that in order to be live on Friday, we'll need to use WP 😞 .

## What does it look like?
screen shot 2017-04-24 at 19 05 31
screen shot 2017-04-24 at 19 04 59

## For interface components
- [ ] Browser testing - Have you checked the component in our supported browsers
- [ ] No javascript - Have you checked the component with javascript disabled
- [ ] A11y testing - Have you run pa11y against the component and fixed/verified all errors, warnings and notices?

## Have the following been considered/are they needed?
- [ ] Tests?
- [ ] Docs?
- [ ] Spoken to the right people?

### CODE REVIEW

File: client/scss/components/_page_description.scss [hunk 1]
```diff
@@ -12,6 +12,20 @@
...
+  @include font('WB7');
+
+  @include respond-to('medium') {
+    @include font('WB6');
+  }
+
+  @include respond-to('large') {
+    @include font('WB5');
+  }
+}
```
Comment (jamesgorrie) [inline client/scss/components/_page_description.scss]: It does feel like we can abstract this type of font setting out, as we do it quite a lot...
Another PR perhaps.

File: server/data/series.js [hunk 2]
```diff
@@ -19,6 +19,17 @@ export function getSeriesCommissionedLength(seriesUrl: string): ?number {
...
+    url: 'body-squabbles',
+    name: 'Body Squabbles',
+    items: List([
+      ({
+        contentType: 'comic',
+        headline: 'Demodicid Navigation',
+        url: ''
+      }: ArticleStub)
+    ])
+  }: Series),
```
Comment (jamesgorrie) [inline server/data/series.js]: Does this not have a `commissionedLength`?

Comment (davidpmccormick) [inline server/data/series.js]: No – it's [effitively infinite](https://github.com/wellcometrust/wellcomecollection.org/issues/663#issuecomment-296601431)

File: server/views/components/image/index.njk [hunk 3]
```diff
@@ -9,4 +10,7 @@ data-srcset="
 {% if data.sizesQueries %}
   sizes="{{ data.sizesQueries }}"
 {% endif %}
-alt="{{ model.caption | striptags() if model.caption }}" />
+{% if model.description %}
+  data-description="{{ model.description }}"
+{% endif %}
+alt="{% if model.alt %} {{ model.alt | striptags() }}{% elif model.caption %} {{ model.caption | striptags() }}{% endif %}" />
```
Comment (jamesgorrie) [inline server/views/components/image/index.njk]: I would say that description makes sense.
Caption is something that normally creates a relationship between the content that the image is in, and the image.

Description is something that is tightly coupled to the image that describes it despite it's context?

File: server/util/body-parser.js [hunk 4]
```diff
@@ -341,7 +341,7 @@ function getImageFromWpNode(node) {
     node.attrs && getAttrVal(node.attrs, 'class') === 'wp-caption-text'
   );
   const alt = getAttrVal(img.attrs, 'alt');
-  const description = getAttrVal(img.attrs, 'data-image-description');
+  const copyright = getAttrVal(img.attrs, 'data-image-description');
```
Comment (jamesgorrie) [inline server/util/body-parser.js]: Hmm. I do wonder if this is the best place for it.
There is a lot of images already in the lib with descriptions, and these will show up.

We could have something along the lines of `getCopyright` that does a check for specific data.

Or we talk to @jtweed's team and see where else we might store this info (but know that we need this soon).

**Pull request merged**